src/Sylius/Bundle/ResourceBundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -16,14 +16,15 @@
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
-use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Sylius\Bundle\TranslationBundle\DependencyInjection\AbstractTranslationExtension;
```

Comment:
Ok, I thought you where saying that the dependency was the other way round.
If we allow translations in the resource bundle it's normal to have a dependency on that bundle.